mappings/hosts.js

```diff
@@ -0,0 +1,25 @@
+var hosts = [{
```

Comment:
Could you please change it to module because we need to import it as a module? I mean something like following:
```javascript
module.exports = [{...}];
```

Reply:
Ok, of course.